shared/django_apps/core/models.py

```diff
@@ -418,6 +418,19 @@ def external_id(self):
         return self.pullid
 
     def should_write_to_storage(self) -> bool:
+        """
+        This only applies to the flare field.
+        Flare is used to draw static graphs (see GraphHandler view in api) and can be large.
+        The majority of flare graphs are used in pr comments, so we keep the (maybe large) flare available while
+        the pull is OPEN. If the pull is not OPEN, we dump the flare to save space. We will recreate it on
+        the fly if it is needed for a non-OPEN pull (see GraphHandler view in api).
+        Flare cleanup is handled by FlareCleanupTask in worker.
+        """
+        if self.state != PullStates.OPEN.value:
+            # while a pull is OPEN, we check whether to write_to_storage
+            # when a pull is no longer OPEN, we no longer want the flare in storage.
+            # The nightly cron job cleans up the value in storage (see FlareCleanupTask in worker)
+            return False
```

Comment:
this would rather mean that the flare is being stored in the JSON field of the table.
I’m unsure what would happen if you change this value based on the `state`. will is copy over data from storage to the database? or will that only happen if you save a new value in the field?

Reply:
good call - I think I was thinking that if this came back `False` it wouldn't be saved anywhere, but you're right, this would just force it onto the db field. Removing.